### DISCUSSION TITLE
Stop rendering as an SVG

### DISCUSSION THREAD

**Discussion by SaminYaser-work** [recharts/recharts#3178] [Q&A] [answered]
Is there a way to render the graphs as a picture or canvas element instead of an SVG? I am working with a large volume of data and rendering is very slow.

**Comment by ckifer** [comment, marked as answer]
Hi @SaminYaser-work - recharts is currently 100% SVG. 

If you look at the code the use of SVG is tightly coupled with everything that recharts does. Maybe someday in the future!
